spec/controllers/collections_controller_spec.rb

```diff
@@ -5,8 +5,8 @@
 RSpec.describe CollectionsController do
   describe 'GET index' do
     it 'assigns @collections ordered by title' do
-      collection1 = Collection.create title: 'zzz collection'
-      collection2 = Collection.create title: 'aaa collection'
+      collection1 = Collection.create title: 'zzz collection', druid: 'druid:abc123', embargo_months: 6, active: true
```

Comment:
can you use the factory here?  `create(:collection)`?

Reply:
Will do.